src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Square {
```

Comment:
The class 'Square' should have properties and methods that define it. Currently, the class is empty. You need to add properties like side length and methods to calculate the area, for example.